src/services/comments.service.ts

```diff
@@ -0,0 +1,33 @@
+import { getAccessToken } from './auth.service'
+import axios from 'axios'
+import { BACKEND_API_URL } from '../config'
+
+export type Comment = {
+  body: string
+  createdAt: string
+  feedbackId: number
+  id: number
+  parentId: number
+  updatedAt: string
+  userId: number
+}
+
+export async function getAllCommentsByFeedbackId(
+  feedbackIds: string
```

Comment:
it could be better to pass an array of ids and then format them into a string here so that if we change approach we change only this particular file